src/relative-time.js

```diff
@@ -0,0 +1,81 @@
+define([
+	"./core",
+	"./common/validate/cldr",
+	"./common/validate/parameter-presence",
+	"./common/validate/parameter-type/string",
+	"./common/validate/parameter-type/number",
+	"./relative-time/format",
+	"./relative-time/properties",
+
+	"./number",
+	"./plural",
+	"cldr/event"
+], function( Globalize, validateCldr, validateParameterPresence, validateParameterTypeString,
+			 validateParameterTypeNumber, relativeTimeFormat, relativeTimeProperties ) {
+
+/**
+ * .formatRelativeTime( value, unit[, options] )
+ *
+ * @value [Number] The number of unit to format.
+ *
+ * @unit [String] eg. "day", "week", "month", etc.
+ *
+ * @options [Object]
+ * - form: [String] eg. "short" or "narrow". Or falsy for default long form
+ * - minWordOffset [Optional Number] The maximum offset when special offset words like
+ *  yesterday and tomorrow will be looked for. Some languages provide several of these.
+ *  default null -> use all available
+ *  Set to 0 to not use any except today, now etc.
```

Comment:
I'm curious why minWordOffset is here, but maxWordOffset is there. I recommend having the options descriptions in one place (there) and having here saying: "see .relativeTimeFormatter() for details"